source/content/migrate-manual.md

```diff
@@ -39,7 +39,7 @@ To ensure a successful migration, complete the following tasks on the source sit
 
 #### .gitignore
 
-Check the contents of your current codebase for existing `.gitignore` files. To be compatible with the platform, using the Pantheon version is advised.  Otherwise, attempts to import files to restricted paths could break the import process. See the platform-provided versions for [WordPress](https://github.com/pantheon-systems/WordPress/blob/default/.gitignore), [Drupal 7](https://github.com/pantheon-systems/drops-7/blob/master/.gitignore), and [Drupal 8](https://github.com/pantheon-systems/drops-8).
+Check the contents of your current codebase for existing `.gitignore` files. To be compatible with the platform, using the Pantheon version is advised. Otherwise, attempts to import files to restricted paths could break the import process. See the platform-provided versions for [WordPress](https://github.com/pantheon-systems/WordPress/blob/default/.gitignore), [Drupal 7](https://github.com/pantheon-systems/drops-7/blob/master/.gitignore), [Drupal 8](https://github.com/pantheon-systems/drops-8), and [Drupal 9](https://github.com/pantheon-systems/drupal-project).
```

Comment:
The Wordpress and D7 URLs go directly to the .gitignore file, but the D8 and D9 URLs just go to the repo.  Should they also link directly to the corresponding .gitignore?
https://github.com/pantheon-systems/drops-8/blob/default/.gitignore
https://github.com/pantheon-systems/drupal-project/blob/default/.gitignore